packages/react-ui/internal/Menu/Menu.styles.ts

```diff
@@ -4,10 +4,15 @@ import { Theme } from '../../lib/theming/Theme';
 const styles = {
   root(t: Theme) {
     return css`
-      background: ${t.bgDefault};
+      background: ${t.menuBgDefault};
       box-sizing: content-box;
       overflow: auto;
-      padding: 5px 0;
+    `;
+  },
+
+  scrollContainer(t: Theme) {
+    return css`
+      padding: ${t.menuPaddingY} 0;
```

Comment:
```suggestion
      padding: ${is8pxTheme(t) ? t.menuPaddingY : 0} 0;
```

Из-за новых паддингов поменялось много скриншотов. Мы льем это в мастер, поэтому все-таки стоит этого избегать. Есть такой способ. Я делал специальный хелпер `is8pxTheme` в `lib/theming/ThemeHelpers.ts`. Он позволяет отличить старую тему от 8px. Используя его, можно применить этот паддинг для разных тем в разных местах, и, тем самым, сохранить скриншоты.